task-sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -540,6 +540,13 @@ class CommsDecoder(Generic[ReceiveMsgType, SendMsgType]):
 
     input: TextIO
 
+    """"
+    It require special case to handle the workloads and api calls to api-server, due to mixing up messages
+    a separate channel is used to send the workloads from parent process to child process the child process.
+    connect_stdin will use this channel to read the workloads in read_workload method.
+    """
```

Comment:
If I remember correctly those should field docstring should go bellow the attr they describe, not above.